### DISCUSSION TITLE
[Simulation question]: Should Type 6 reactions be scaled for temperature?

### DISCUSSION THREAD

**Discussion by jonahpeter** [topas-nbio/TOPAS-nBio#17] [Simulation-question] [answered]
### Simulation question

I am working to extend the temperature scaling for my needs. A fundamental question I have is whether Type 6 reactions should be scaled for temperature. Since the scavenging capacity is just the reaction constant times the background concentration, I would have thought that these reactions should inherit the same temperature dependence of the reaction constant.

However, in `TsIRTConfiguration::AdjustReactionAndDiffusionRateForTemperature()` there is a block of code that reads
```
if (Found) {
			if (type != 6) {
				if (newKobs > 0)
					fReactions[i].kobs = newKobs * fPm->GetUnitValue("/M/s");
				if (newKact > 0)
					fReactions[i].kact = newKact * fPm->GetUnitValue("/M/s");
				if (newKdif > 0)
					fReactions[i].kdif = newKdif * fPm->GetUnitValue("/M/s");
			}
		}
```
which means Type 6 reactions _don't_ get scaled for temperature, even though there are two Type 6 equivalents within the hard-coded set of reactions.

Is there a reason why Type 6 reactions shouldn't be scaled for temperature?

### TOPAS-nBio category

Chemistry

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment, marked as answer]
Hi there,
Those also should be scaled by temperature, but you don't need type VI for G-values within 1 µs at low LET radiation, which are the conditions of the TOPAS-nBio implementation. If you find/have rates scaled, the you can unlock that conditional loop and add your values for type VI.

**Reply by jonahpeter** [reply]
thanks!
